packages/app/src/utils/const.ts

```diff
@@ -9,6 +9,10 @@ export const ANYONE_CAN_PAY_TESTNET_TX_HASH = '0x4f32b3e39bd1b6350d326fdfafdfe05
 export const SECP256K1_BLAKE160_CODE_HASH = '0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8'
 export const SECP256K1_BLAKE160_TESTNET_TX_HASH = '0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37'
 export const SECP256K1_BLAKE160_MAINNET_TX_HASH = '0x71a7ba8fc96349fea0ed3a5c47992e3b4084b031a42264a018e0072e8172e46c'
+// DEVENT_TX_HASH custom by yourself
```

Comment:
Actually, devnet is for users to import custom scripts, including the default lock script. So it could have no strong binding to its tx hash.

Otherwise, once the app is packaged and distributed, users will have an invalid tx hash when they switch to devnet locally.

Say, your devnet has secp256k1 tx hash `A` and my devnet has secp256k1 tx hash `B`. And when I use your packaged app, the default tx hash is missing in my local devnet.